datasets/timit_asr/timit_asr.py

```diff
@@ -129,14 +129,18 @@ def _split_generators(self, dl_manager):
     def _generate_examples(self, split, data_dir):
         """Generate examples from TIMIT archive_path based on the test/train csv information."""
         # Iterating the contents of the data to extract the relevant information
-        for wav_path in sorted(Path(data_dir).glob(f"**/{split.upper()}/**/*.WAV")):
+        wav_paths = sorted(Path(data_dir).glob(f"**/{split.upper()}/**/*.wav"))
+        wav_paths = wav_paths if wav_paths else sorted(Path(data_dir).glob(f"**/{split.upper()}/**/*.WAV"))
```

Comment:
This is fine if it can't have both .WAV and .wav files in the same folder - are we sure about this ?

Reply:
@lhoestq no, we are not sure... We need to have the data from LDC.

Otherwise, I could concatenate both globs, but I'm afraid it could give twice the filenames on Windows machines...

Reply:
I think it's the same data on deepai than on LDC. I've only used the data from deepai to update the PR.

We can mergi this one for now and ask the user who has the data from LDC to run it and report if there's any issue I guess